sandboxes/dynamicanalysis/Dockerfile

```diff
@@ -76,9 +81,7 @@ RUN php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');" &&
 # NPM setup
 #
 WORKDIR /setup/node
-RUN apt-get update && apt-get -y upgrade && apt-get install -y --no-install-recommends \
-	nodejs \
-	npm
+RUN apt-get update && apt-get install -y --no-install-recommends nodejs
```

Comment:
nit: comment that NPM is pulled in as a dependency of nodejs

Reply:
Does this help?

Reply:
yes, thanks!